### ISSUE TITLE
[0.9.0.0 beta staging-1732] Law : Property Transfer missing trigger options and statement options

### ISSUE THREAD

**Issue by IcedForge** [StrangeLoopGames/EcoIssues#17813]
Missing options for "Property Transfer" Trigger 
Tooltip
![Screenshot_60](https://user-images.githubusercontent.com/24981362/90789336-17b16780-e307-11ea-8a5a-1010766b780f.jpg)

We are missing option for Location in Trigger selection or change/add another IF statement specific to property transfer to clarify ( check law example for more info ).
![Screenshot_61](https://user-images.githubusercontent.com/24981362/90789437-36aff980-e307-11ea-9671-614860789e85.jpg)

Also missing a "Citizen who received the property" value to add conditioning on the recipient not just the seller.

![Screenshot_59](https://user-images.githubusercontent.com/24981362/90789462-3fa0cb00-e307-11ea-983b-5dada46962bf.jpg)

--------------------
Law Example
Trying to create a law prohibiting someone buying/receiving a property if he already owns one inside a specific district(Federal Capital) but the conditioning is only set if the action is located?.

This will now perform a check on the user who is selling; which he clearly owns 1 or more claims due to him wanting to sell a claimed area.
As there is no way to add a check for a "Citizen who receive property" this is impossible as far as i can tell?
![Screenshot_63](https://user-images.githubusercontent.com/24981362/90790805-c0ac9200-e308-11ea-8234-1a65329d27c3.jpg)

Using If - District requirement makes the law description very non user friendly as it states 
![Screenshot_62](https://user-images.githubusercontent.com/24981362/90790151-0c127080-e308-11ea-85be-561dcef60afa.jpg)

Which makes one think that the action of the sale would only be restricted inside that district and it would be fine to sell/transfer it if you are located outside that district not that the property itself  IS inside that zoning.

**Comment by beastyn** [comment]
@IcedForge actually I was talking about it with John, and its a bit frustrating. These properties in a tooltip do not actually go to the trigger options. They can be used in IF section. Like Location property will give you Location category in IF section (like District Requirement)

So, if I got it right:
We want to:
1) Get citizen who is receiving property (we actually should from tooltip info, but I did not find it anywhere)
2) Get more clear description from IF section (I do not see this part about Property Owned checks)

**Comment by IcedForge** [comment]
Exacly that would be correct beastyn, at least from a player point of view it would help a lot and make it a little bit more friendly to use.

**Comment by johnkslg** [comment]
Fixed this. The issue was that property could be owned by an 'alias' (that means a user, demographic, or title) so it couldnt fit into the 'user' slots of certain variables.  I've changed it so you can do this now:
![image](https://user-images.githubusercontent.com/3536496/95289324-518aee80-081f-11eb-851c-a44272f1d6e9.png)

I also fixed the Ignore Auth to work properly for claiming and unclaiming, previously it wouldnt let you remove auth for that action. 

So with this law, if as an admin I go unclaim some non-admin's land I get this in chat: 
![image](https://user-images.githubusercontent.com/3536496/95289384-71221700-081f-11eb-88fc-1e946988d78e.png)

However, this opens weirdness because demographics and titles can own land.
![image](https://user-images.githubusercontent.com/3536496/95289473-a0d11f00-081f-11eb-8ff8-da3a0d951dde.png)

![image](https://user-images.githubusercontent.com/3536496/95289455-9878e400-081f-11eb-912b-28b678ce6be7.png)

So is this 'Abandoned' property owner an admin in the law or 'not an admin'?  I've gone with the solution that it is *neither* an admin nor not an admin.  

This will take some testing for other conditions and legal actions that have user slots, qa please go through and test lots of cases of using context values that resolve to a non-user in slots that are traditionally users. IE, try to get the bank account of a demographic/title, try to get their wealth, skill level, etc.  As far as I know it should all work (they should all fail as conditions, ie return false) but it's worth checking. 

For the second part of the issue about the location not being clear, I added a static tooltip that will show up, but tooltips are blocked on another issue at the moment, and when that is fixed it should automatically appear.
